expression/builtin_math.go

```diff
@@ -1474,11 +1474,12 @@ type truncateFunctionClass struct {
 	baseFunctionClass
 }
 
+// getDecimal return the `Decimal` value of return type for function `TRUNCATE`
 func (c *truncateFunctionClass) getDecimal(sc *variable.StatementContext, arg Expression) int {
 	if constant, ok := arg.(*Constant); ok {
 		decimal, isNull, err := constant.EvalInt(nil, sc)
 		if err != nil {
-			log.Errorf("truncateFunctionClass.getDecimal with error: %v", err.Error())
+			log.Warningf("truncateFunctionClass.getDecimal() with error: %v", err.Error())
```

Comment:
This will write a lot of logs, may be we do not need this line.
``` sql
create table t(a char);
insert 10w records into t;
select truncate("abcd", a);
```